tools/syz-testbuild/testbuild.go

```diff
@@ -125,24 +124,26 @@ func main() {
 }
 
 func test(repo vcs.Repo, bisecter vcs.Bisecter, kernelConfig []byte, env instance.Env, com *vcs.Commit) {
-	compiler, compilerType, linker, ccache := "gcc", "gcc", "ld", ""
+	compiler, compilerType, linker, ccache, makeBin := "gcc", "gcc", "ld", "", "make"
```

Comment:
`Make` is anyway `make` by default, right?
Unless we want to add some `flagMake` above, it seems unnecessary to explicitly set it here.